calculator.js

```diff
@@ -1,3 +1,8 @@
+// var myCalculator = calculatorModule(); // creates a new calculator to be used
+//     myCalculator.load(6); // initial value to be stored in a private variable named `total`
+//     myCalculator.add(5); // adds 5 to `total`
+//     myCalculator.getTotal(); // returns the value of `total`
+
```

Comment:
Please remove unused comments instead of committing commented out lines of code.